tests/test_translation_display.py

```
@@ -0,0 +1,73 @@
+import pytest
+from translate_srt import TranslationDisplay, MODEL_PRICING
+
+@pytest.fixture
+def display():
```

Ruff (0.8.2)

Missing return type annotation for public function `display`

(ANN201)